packages/example-codehub/src/controllers/health-controller.api.ts

```diff
@@ -3,18 +3,28 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-export const def = {
+import {OpenApiSpec} from '@loopback/core';
+
+export const def: OpenApiSpec = {
+  swagger: '2.0',
+  info: {
+    title: 'unused',
+    version: 'unused',
+  },
```

Comment:
Good points! I thought we want to use full OpenAPI spec for controllers, which does not make that much sense on the second thought. I'll create a new interface as proposed by Simon.